docs/reference/operator_parameters.md

```diff
@@ -377,6 +377,9 @@ key.
   List of roles that cannot be overwritten by an application, team or
   infrastructure role. The default is `admin`.
 
+* **postgres_admin_teams**
```

Comment:
This is confusing. We give "admin" to owning teams and call the superusers admins. Let's call it by what is is. postgres_superuser_teams?